www/main/settings/components/Settings.jsx

```diff
@@ -5,15 +5,22 @@ import { Overlay } from '../../common/sttm-ui';
 
 import SettingsNav from './SettingsNav';
 import SettingsContainer from './SettingsContainer';
-
+import ThemeContainer from './ThemeContainer';
+import SettingViewer from './SettingViewer';
 import { settingsNavObj, settingsObj } from '../utils';
 
 const Settings = ({ onScreenClose }) => {
```

Comment:
Think we need to improve the CSS for smaller view ports 
![image](https://user-images.githubusercontent.com/2982784/110660496-7b529380-8178-11eb-835e-405287a67596.png)